### PR TITLE
fix: add matthewraymondh

### LINKED ISSUE #933: Matthew Raymond
## Profile Screenshot

screencapture-github-matthewraymondh-2021-12-22-21_26_37

https://user-images.githubusercontent.com/83885819/147108033-04ee7975-97ac-421c-96cc-7330603ed1dd.png

### PR DESCRIPTION
**Pull request by matthewraymondh**
resolves #933

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/matthewraymondh.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/83885819/147108033-04ee7975-97ac-421c-96cc-7330603ed1dd.png",
```
Comment (Panquesito7) [inline profiles/matthewraymondh.json:2]: I recommend more of an image like this.

image

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/147137654-7f453c7e-b6f5-4b68-bcfc-fd9c0a104c04.png",
```

File: profiles/matthewraymondh.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  "issueId": 930,
```
Comment (Panquesito7) [inline profiles/matthewraymondh.json:3]:
```suggestion
  "issueId": 933,
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by matthewraymondh** [comment]
Done !

File: profiles/matthewraymondh.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "image": "https://user-images.githubusercontent.com/83885819/147108033-04ee7975-97ac-421c-96cc-7330603ed1dd.png",
```
Comment (krishguptadev) [inline profiles/matthewraymondh.json]:
```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/147137654-7f453c7e-b6f5-4b68-bcfc-fd9c0a104c04.png",
```

Comment (matthewraymondh) [reply profiles/matthewraymondh.json]: i commited the same suggestion from panquesito...

Comment (krishguptadev) [reply profiles/matthewraymondh.json]: You didn't commit it 😅

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**